doc/metrics.md

```diff
@@ -0,0 +1,65 @@
+# Instance metrics
+LXD provides metrics for all running instances. Those covers CPU, memory, network, disk and process usage and are meant to be consumed by Prometheus and likely graphed in Grafana.
+In cluster environments, LXD will only return the values for instances running on the server being accessed. It's expected that each cluster member will be scraped separately.
+The instance metrics are updated when calling the `/1.0/metrics` endpoint.
+They are cached for 15s to handle multiple scrapers. Fetching metrics is a relatively expensive operation for LXD to perform so we would recommend scraping at a 30s or 60s rate to limit impact.
+
+# Create metrics certificate
+The `/1.0/metrics` endpoint is a special one as it also accepts a `metrics` type certificate.
+This kind of certificate is meant for metrics only, and won't work for interaction with instances or any other LXD objects.
+
+Here's how to create a new certificate (this is not specific to metrics):
+
+```bash
+openssl req -x509 -newkey rsa:2048 -keyout ~/.config/lxc/metrics.key -nodes -out ~/.config/lxc/metrics.crt -subj "/CN=lxd.local"
+```
+
+Now, this certificate needs to be added to the list of trusted clients:
+
+```bash
+lxc config trust add ~/.config/lxc/metrics.crt --type=metrics
+```
+
+# Add target to prometheus
+In order for prometheus to scrape from LXD, it has to be added to the targets.
+
+First, one needs to ensure that core.https_address is set so LXD can be reached over the network.
+This can be done by running:
+
+```bash
+lxc config set core.https_address ":8443"
+```
+
+Second, the newly created certificate and key, as well as the LXD server certificate need to be accessible to prometheus.
```

Comment:
Prometheus should probably have a capital "P" everywhere you mention it in the docs.